targets/TARGET_STM/TARGET_STM32F4/TARGET_UBLOX_C030/device/TOOLCHAIN_IAR/stm32f439xx.icf

```diff
@@ -0,0 +1,44 @@
+/*###ICF### Section handled by ICF editor, don't touch! ****/
+/*-Editor annotation file-*/
+/* IcfEditorFile="$TOOLKIT_DIR$\config\ide\IcfEditor\cortex_v1_0.xml" */
+/*-Specials-*/
+define symbol __ICFEDIT_intvec_start__ = 0x08000000;
+/*-Memory Regions-*/
+define symbol __ICFEDIT_region_ROM_start__     = 0x08000000;
+define symbol __ICFEDIT_region_ROM_end__       = 0x080FFFFF;
+define symbol __ICFEDIT_region_NVIC_start__    = 0x20000000;
+define symbol __ICFEDIT_region_NVIC_end__      = 0x200001AF;
+define symbol __ICFEDIT_region_RAM_start__     = 0x200001B0;
+define symbol __ICFEDIT_region_RAM_end__       = 0x2002FFFF;
+define symbol __ICFEDIT_region_CCMRAM_start__  = 0x10000000;
+define symbol __ICFEDIT_region_CCMRAM_end__    = 0x1000FFFF;
+define symbol __ICFEDIT_region_BKPSRAM_start__ = 0x40024000;
+define symbol __ICFEDIT_region_BKPSRAM_end__   = 0x40024FFF;
+/*-Sizes-*/
+/*Heap 1/2 of ram and stack 1/8*/
+define symbol __ICFEDIT_size_cstack__ = 0x6000;
```

Comment:
Frankly, I just copied the ODIN one as the processor is the same RAM-wise.  What size would you suggest for the needs of mbed-os?